src/wallet/rpcwallet.cpp

```diff
@@ -3567,7 +3567,7 @@ UniValue rescanblockchain(const JSONRPCRequest& request)
     }
```

Comment:
Is there a reason you didn't remove the `start_block` variable?

Reply:
i didn't want to change the function called but i guess i should

Reply:
I think this can be removed now that block hash is gone here.

Reply:
indeed